mobify/nightwatch-commands
# Update linting to current standards

Status: **Ready for Review**
Owner: @marlowpayne
Reviewers: @mobify-derrick @ellenmobify 

## Changes
- Updates project to use current JS linting standards (eslint + updated code style rulesets)
- Lints a lot more of the project's code than before
- Fix _allll_ the current lint errors
- Lint the project code as part of `npm test` (will fix tests in separate PR + integrate with CircleCI)

## Todos:
- [x] QA +1

### Feedback:
_none so far_

## How to Test
- Checkout code
- Run `grunt lint` to check the linter
- Make a JS syntax error and ensure the linter catches it
- Run `[sudo] npm link` and make sure the custom assertions/commands still work in other projects
